### PR TITLE
#36 SqlOrchestrationService.GetManyOrchestrationsAsync - convert spro…

### PR DESCRIPTION
**Pull request by usemam**
Attempt on addressing #36
SqlOrchestrationService.GetManyOrchestrationsAsync - convert sproc datetime parameters to UTC

### CODE REVIEW

File: src/DurableTask.SqlServer/SqlUtils.cs [hunk 1]
```diff
@@ -445,10 +445,13 @@ public static SqlDateTime ToSqlDateTime(this DateTime dateTime, SqlDateTime defa
             {
                 return SqlDateTime.MinValue;
             }
-            else
+            else if (dateTime.Kind == DateTimeKind.Utc)
             {
                 return dateTime;
             }
+            else {
```
Comment (cgillum) [inline src/DurableTask.SqlServer/SqlUtils.cs:452]: Let's put this `{` on its own line for code-style consistency.

**Review by cgillum — COMMENTED** [review]
Thanks for this PR! One small nit. Also, I think we'll need to check to see whether the CI test failure is an actual failure or if it's flakiness.

**Comment by usemam** [comment]
Looks like it was a flake after all. All green now.

**Review by cgillum — APPROVED** [review]
Great - thanks again for this PR!

**Pull request merged**